app/src/main/java/net/aiscope/gdd_app/ui/mask/MaskActivity.kt

```diff
@@ -57,38 +58,42 @@ class MaskActivity : AppCompatActivity(), MaskView, CaptureFlow {
 
         presenter.start(diseaseName, imageNameExtra)
 
-        if (savedInstanceState == null) {
-            setBrushDrawableDiseaseStage(brushDiseaseStages[0])
-        }
+        currentDiseaseStage =
```

Comment:
I think I wanted to keep consistency, as the other components used to need the whole object, not just the color. Now they don't need it anymore, so I'm changing all of them as per your suggestion. Thanks!!

Reply:
Err... There's a problem with this actually - currentDiseaseStage is a lateinit, because it comes from the presenter, which is an injected lateinit, and I you can't have a primitive with lateinit... Any tips to get past through this?

Reply:
Would initialising it with `0` work?

Reply:
Yes, I just resisted the idea because it didn't seem very clean to me... But if you think that's fine, doing! Thanks!